doc/news/changes/minor/20210220DavidWells

```diff
@@ -0,0 +1,5 @@
+Fixed: deal.II should now be able to use versions of LAPACK compiled with
+Fortran compilers that do not adhere to the standard Fortran mangling
+convention.
```

Comment:
I don't think there actually *is* a standard naming convention. Maybe use "usual" instead?

Reply:
Good idea. I fixed it.